packages/kit/postinstall.js

```diff
@@ -0,0 +1,27 @@
+import fs from 'fs';
+import path from 'path';
+import glob from 'tiny-glob/sync.js';
+import { load_config } from './src/core/config/index.js';
+import * as sync from './src/core/sync/sync.js';
+
+const pkg = JSON.parse(fs.readFileSync('package.json', 'utf8'));
+
+const directories = [];
+
+const cwd = process.env.INIT_CWD ?? process.cwd();
+
+if (pkg.workspaces) {
+	for (const directory of pkg.workspaces) {
+		directories.push(...glob(directory, { cwd }).map((dir) => path.resolve(cwd, dir)));
+	}
+} else {
+	directories.push(cwd);
+}
+
+for (const cwd of directories) {
+	if (fs.existsSync(path.join(cwd, 'svelte.config.js'))) {
+		process.chdir(cwd);
+		const config = await load_config();
+		await sync.all(config, 'development');
```

Comment:
While we're at it: I noticed that `npm install ` fails when you have an invalid `svelte.config.js`. This makes it very inconvenient for (for example) people upgrading from previous versions where some configs are now obsolete - for example, you update to the new form actions and used method overrides previously, then you first need to remove that from the config before `npm install`. It would probably be better to only show a warning in this case and continue installing the package. The error will still prevent you from actually starting the app - which is good - but you get a successful install - which is also good.

```suggestion
		try {
			const config = await load_config();
			await sync.all(config, 'development');
		} catch (e) {
			console.log('Error while trying to sync SvelteKit config');
			console.log(e);
		}
```